instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/_client.py

```diff
@@ -59,12 +62,27 @@ def callback(response_future):
     return callback
 
 
+def _safe_invoke(function: Callable, *args):
+    function_name = "<unknown>"
+    try:
+        function_name = function.__name__
```

Comment:
I think this function name is redundant because it'll probably be called "request_hook" :)